docs/guides/development/graphql/graphql-apollo-an-introduction/index.md

```diff
@@ -0,0 +1,230 @@
+
+## What is Apollo GraphQL?
+
+As a public standard, the GraphQL language admits multiple implementations.  [Apollo](https://www.apollographql.com/) maintains the most prominent of these implementations, with high-profile [venture capital backing](https://www.crunchbase.com/organization/meteor) and headquarters in San Francisco.  Apollo’s implementation is largely [open-source](https://www.apollographql.com/docs/apollo-server/):  anyone can use it freely.  Apollo makes money on a [hosted infrastructure](https://www.apollographql.com/blog/announcement/introducing-apollo-engine-insights-error-reporting-and-caching-for-graphql-6a55147f63fc/) that brings value to large-scale applications of GraphQL, as well as other more specialized services.
+
+Be clear on these distinctions:  GraphQL is a language defined outside Apollo.  Apollo GraphQL is the collection of implementations the Apollo company bases on the GraphQL standard.
+
+## What is the Apollo GraphQL Client?
+
+The initial GraphQL example above is about the name of a particular individual.  In the GraphQL model, such data is sent between a *client* and *server*:  the client issues a request to the server, and the server answers the client’s request.  The Apollo GraphQL client is open-source, freely-available software which expresses GraphQL queries, transmits them to a GraphQL server, and receives responses.  Because GraphQL is a public standard, it is possible to use an Apollo GraphQL client with any GraphQL server; the latter doesn't have to be from Apollo.
+
+### Apollo GraphQL Client Example
+
+While Apollo provides extensive [documentation](https://www.apollographql.com/docs/), including high-quality [tutorials](https://www.apollographql.com/docs/tutorial/introduction/), this example is simpler, more direct, and more current than those in the Apollo tutorials.  Recall that GraphQL always communicates between a client and server.  Your first example client takes advantage of an open [GraphQL service](https://api.spacex.land/graphql/) that [SpaceX](https://www.spacex.com/) provides.  This allows you to focus entirely on the client’s side of communications.
+
+All the configuration which follows assumes a good installation of [npm](https://www.npmjs.com/) and [Node.js](https://nodejs.org/en/).
+
+From a convenient command line, request
+
+    npm install graphql @apollo/client cross-fetch
+
+This should take only a few seconds to complete.
+
+Create `client-example.mjs` with the following content:
+
+{{< file "client-example.mjs" >}}
+// cross-fetch helps ensure correct behavior whatever environment
+// needs the HttpLink.
+import fetch from 'cross-fetch';
+import {
+  ApolloClient, gql, HttpLink, InMemoryCache
+} from "@apollo/client";
+
+const cache = new InMemoryCache()
+const uri = 'https://api.spacex.land/graphql/';
+const query = gql`
+  query {
+    launchesPast(limit: 10) {
+      mission_name
+    }
+  }
```

Comment:
This line is still missing the backtick I mentioned earlier.  The line should be

    }`

Reply:
Hello @claird, thanks for your comments. I have added the missing backtick you mentioned. When I run the `node client-example.mjs` file, I get the following error: